deps/pom.xml

```diff
@@ -0,0 +1,33 @@
+<!--
+  Licensed to the Technische Universität Darmstadt under one
+  or more contributor license agreements. See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership. The Technische Universität Darmstadt
+  licenses this file to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <parent>
+        <groupId>de.tudarmstadt.ukp.wikipedia</groupId>
+        <artifactId>de.tudarmstadt.ukp.wikipedia</artifactId>
+        <version>2.0.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>deps</artifactId>
```

Comment:
IMHO artifact names should all share a common prefix like e.g. `jwpl`.

Reply:
I have no objections in changing to `dkpro-jwpl-XXX`. WDYT @mawiesne ?

Reply:
And while at it, the groupId could also be updated to `org.dkpro.jwpl`.

Reply:
(e.g. DKPro Core as groupId `org.dkpro.core` for some time now)

Reply:
see: https://github.com/dkpro/dkpro-jwpl/issues/165

Reply:
I agree. we could do the relocation / renaming as proposed in #165, but separately.

Reply:
Yeah - let's tackle this for 2.0.0 but in another PR to ease reviewer burden :)